core/src/com/mygdx/crazybomber/LoadScreen.java

```diff
@@ -0,0 +1,73 @@
+//TODO: fade in & out before main menu
+
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+
+public class LoadScreen implements Screen {
+    private Sprite background;
+    private SpriteBatch batch;
+    @Override
+    public void show() {
+        StateRepository stateRepository = new StateRepository();
```

Comment:
Since we have a repository, im thinking if we should have a serviceLocator which would hold all the data that needs to be passed into each state... You should probably worry about this comment last, ill help you with it once im back.